packages/edit-post/src/components/layout/index.js

```diff
@@ -87,45 +87,52 @@ function Layout( { styles } ) {
 
-		return {
-			isTemplateMode: select( editPostStore ).isEditingTemplate(),
-			hasFixedToolbar:
-				select( editPostStore ).isFeatureActive( 'fixedToolbar' ),
-			sidebarIsOpened: !! (
-				select( interfaceStore ).getActiveComplementaryArea(
-					editPostStore.name
-				) || select( editPostStore ).isPublishSidebarOpened()
-			),
-			isFullscreenActive:
-				select( editPostStore ).isFeatureActive( 'fullscreenMode' ),
-			isInserterOpened: select( editPostStore ).isInserterOpened(),
-			isListViewOpened: select( editPostStore ).isListViewOpened(),
-			mode: select( editPostStore ).getEditorMode(),
-			isRichEditingEnabled: editorSettings.richEditingEnabled,
-			hasActiveMetaboxes: select( editPostStore ).hasMetaBoxes(),
-			previousShortcut: select(
-				keyboardShortcutsStore
-			).getAllShortcutKeyCombinations( 'core/edit-post/previous-region' ),
-			nextShortcut: select(
-				keyboardShortcutsStore
-			).getAllShortcutKeyCombinations( 'core/edit-post/next-region' ),
-			showIconLabels:
-				select( editPostStore ).isFeatureActive( 'showIconLabels' ),
-			isDistractionFree:
-				select( editPostStore ).isFeatureActive( 'distractionFree' ) &&
-				isLargeViewport,
-			showBlockBreadcrumbs: select( editPostStore ).isFeatureActive(
-				'showBlockBreadcrumbs'
-			),
-			// translators: Default label for the Document in the Block Breadcrumb.
-			documentLabel: postTypeLabel || _x( 'Document', 'noun' ),
-		};
-	}, [] );
+			return {
+				isTemplateMode: select( editPostStore ).isEditingTemplate(),
+				hasFixedToolbar:
+					select( editPostStore ).isFeatureActive( 'fixedToolbar' ),
+				sidebarIsOpened: !! (
+					select( interfaceStore ).getActiveComplementaryArea(
+						editPostStore.name
+					) || select( editPostStore ).isPublishSidebarOpened()
+				),
+				isFullscreenActive:
+					select( editPostStore ).isFeatureActive( 'fullscreenMode' ),
+				isInserterOpened: select( editPostStore ).isInserterOpened(),
+				isListViewOpened: select( editPostStore ).isListViewOpened(),
+				mode: select( editPostStore ).getEditorMode(),
+				isRichEditingEnabled: editorSettings.richEditingEnabled,
+				hasActiveMetaboxes: select( editPostStore ).hasMetaBoxes(),
+				previousShortcut: select(
+					keyboardShortcutsStore
+				).getAllShortcutKeyCombinations(
+					'core/edit-post/previous-region'
+				),
+				nextShortcut: select(
+					keyboardShortcutsStore
+				).getAllShortcutKeyCombinations( 'core/edit-post/next-region' ),
+				showIconLabels:
+					select( editPostStore ).isFeatureActive( 'showIconLabels' ),
+				isDistractionFree:
+					select( editPostStore ).isFeatureActive(
+						'distractionFree'
+					) && isLargeViewport,
+				showBlockBreadcrumbs: select( editPostStore ).isFeatureActive(
+					'showBlockBreadcrumbs'
+				),
+				// translators: Default label for the Document in the Block Breadcrumb.
+				documentLabel: postTypeLabel || _x( 'Document', 'noun' ),
+			};
+		},
+		[ isLargeViewport ]
```

Comment:
Same question as for https://github.com/WordPress/gutenberg/pull/47611/files#r1091830264